File: src/main/Java/boj_step13/boj1427.java
import java.io.*;
import java.util.Arrays;
import java.util.Collections;

public class boj1427 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        char[] line = reader.readLine().toCharArray();
        Integer[] list = new Integer[line.length];
        for (int i = 0; i < line.length; i++) list[i] = Character.getNumericValue(line[i]);
        Arrays.sort(list, Collections.reverseOrder());
        for (int it: list) writer.append(String.valueOf(it));
        writer.close();
    }
}

File: src/main/boj_step13/boj1427.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter
import java.time.temporal.ValueRange

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val n = r.readLine()
    val c = n.toCharArray()
    val nArray = IntArray(c.size)
    c.forEachIndexed { index, it ->
        nArray[index] = it - '0'
    }
    nArray.sort()
    nArray.reverse()
    nArray.forEach { w.write(it.toString()) }
    w.close()
}